src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -120,18 +120,18 @@ export const fetchDictionaryConcepts = (
   page = 1,
 ) => async (dispatch, getState) => {
   dispatch(isFetching(true));
-  let url = `${conceptType}/${conceptOwner}/sources/${conceptName}/concepts/?includeMappings=true&q=${query}&limit=${limit}&page=${page}&verbose=true`;
+  let url = `${conceptType}/${conceptOwner}/collections/${conceptName}/concepts/?includeMappings=true&q=${query}&limit=${limit}&page=${page}&verbose=true`;
```

Comment:
Are these changes also part of pagination?

Reply:
I pulled master but I didn't realise these changes during the merge. Let me make a change.